src/components/DataTable/DataTable.tsx

```diff
@@ -125,6 +127,8 @@ const DataTable = <T extends { id: number; selectDisabled?: boolean }>({
   emptyTitle,
   emptySubtitle,
   per,
+  defaultSortField,
+  defaultSortOrder,
```

Comment:
今だと初期値がuseOrderStateに依存してるので、
```suggestion
  defaultSortOrder = "desc",
```
とした方が読みやすそう

Reply:
たしかにそうですね！修正します